src/PositionManager.sol

```diff
@@ -50,7 +53,7 @@ contract PositionManager is
     uint256 public nextTokenId = 1;
 
     /// @inheritdoc IPositionManager
-    mapping(uint256 tokenId => bytes32 configId) public positionConfigs;
+    mapping(uint256 tokenId => bytes32 config) public positionConfigs;
```

Comment:
should this be private? now that the id is a mixture of information? and have a getter or 2?